CHANGES.rst

```diff
@@ -11,6 +11,7 @@ Improvements
 ^^^^^^^^^^^^
 
 - Use the revision's timestamp when downloading its files as a ZIP archive (:pr:`5686`)
+- Replaced the colored status button with a dot on the judgment timeline (:issue:`5687`, :pr:`5697`)
```

Comment:
```suggestion
- Use more consistent colors on the editing judgment button (:issue:`5687`, :pr:`5697`)
```

- this is the relevant part; how it's done is more of a detail
- we use present tense in the changelog (just like in commit messages)